File: executable/r/text-chart-basic.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(0, 1, 2), 
  y = c(1, 1, 1), 
  text = c("Text A", "Text B", "Text C"), 
  textposition = "top", 
  name = "Lines, Markers and Text", 
  mode = "lines+markers+text", 
  type = "scatter"
)
trace2 <- list(
  x = c(0, 1, 2), 
  y = c(2, 2, 2), 
  text = c("Text D", "Text E", "Text F"), 
  textposition = "bottom", 
  name = "Markers and Text", 
  mode = "markers+text", 
  type = "scatter"
)
trace3 <- list(
  x = c(0, 1, 2), 
  y = c(3, 3, 3), 
  text = c("Text G", "Text H", "Text I"), 
  textposition = "bottom", 
  name = "Lines and Text", 
  mode = "lines+text", 
  type = "scatter"
)

layout <- list(showlegend = FALSE)



response <- p$plotly(trace0, trace1, trace2, kwargs=list(layout=layout, filename="text-chart-basic", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/python/text_chart_basic.py
# Learn about API authentication here: {{BASE_URL}}/python/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')
trace1 = Scatter(
    x=[0, 1, 2],
    y=[1, 1, 1],
    mode='lines+markers+text',
    name='Lines, Markers and Text',
    text=['Text A', 'Text B', 'Text C'],
    textposition='top'
)
trace2 = Scatter(
    x=[0, 1, 2],
    y=[2, 2, 2],
    mode='markers+text',
    name='Markers and Text',
    text=['Text D', 'Text E', 'Text F'],
    textposition='bottom'
)
trace3 = Scatter(
    x=[0, 1, 2],
    y=[3, 3, 3],
    mode='lines+text',
    name='Lines and Text',
    text=['Text G', 'Text H', 'Text I'],
    textposition='bottom'
)
data = Data([trace1, trace2, trace3])
layout = Layout(
    showlegend=False
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='text-chart-basic', auto_open=False)
